rialto_airflow/dags/harvest.py

```diff
@@ -123,9 +124,18 @@ def distill_publications(snapshot, openalex_fill_in, dimensions_fill_in):
         """
         Distill the publication metadata into publication table columns.
         """
-        distill.distill(snapshot)
+        count = distill.distill(snapshot)
```

Comment:
We currently need to return something, in order to get the tasks dependencies to wire up correctly. But I thought it might make more sense to return the number of changes made instead of the snapshot.